src/LinuxTracing/PerfEvent.h

```diff
@@ -249,6 +256,48 @@ struct DmaFenceSignaledPerfEventData {
 };
 using DmaFenceSignaledPerfEvent = TypedPerfEvent<DmaFenceSignaledPerfEventData>;
 
+// This struct is supposed to resemble the perf_record_sample, all commented out
+// fields are fields we don't currently use anywhere.
```

Comment:
Nit: You should also refer to the Consume function.

Reply:
Updated the comment after moving the struct.